### ISSUE TITLE
How to adjust height of WYSIWYG fields in 2.1.1?

### ISSUE THREAD

**Issue by peterboorsma** [bolt/bolt#3147]
In earlier versions of Bolt one could alter the height of a WYSIWYG field in contenttypes.yml.
Like this:
        body:
            type: html
            height: 600px

But not in The latest Bolt versions. Has something changed in the markup? Or is it just b0rken?

**Comment by rarila** [comment]
Looks like for now you have to do it this way: #1782

I will have a look into this and add setting it directly from the height attribute later hitting in master.

**Comment by bobdenotter** [comment]
The method described in #1782 doesn't work (anymore). :-)

**Comment by rarila** [comment]
Because it has to be:
```
options:
                ckeditor:
                    height: 123
```

**Comment by bobdenotter** [comment]
I really can't get it to work. Are you sure?

**Comment by rarila** [comment]
You have to also set the autoGrow values!

**Comment by GDmac** [comment]
Or set ``autoGrow_onStartup: false``
(see html field description/example)
https://docs.bolt.cm/contenttypes-and-records

Alternatively, we could have the html field ``height`` setting be some  
kind of macro to automatically set options:height and autogrow:false

**Comment by bobdenotter** [comment]
Closed by #3156
